schema/test/unit/computed_columns/project_revision_project_manager_form_changes_by_label_test.sql

```diff
@@ -52,15 +52,21 @@ insert into cif.project(id, operator_id, funding_stream_rfp_id, project_status_i
     (5, 1, 1, 1, '005', 'summary', 'project 1'),
     (6, 1, 1, 1, '006', 'summary', 'project 1');
 
-insert into cif.project_revision(id, change_status, change_reason, project_id)
+insert into cif.revision_type (type)
+  values
+    ('Amendment'),
+    ('General Revision'),
+    ('Minor Revision');
```

Comment:
These should be set by the migration, do we truncate them?

Reply:
I think it's getting truncated as part of a cascade? If I don't include these here I get this error: `DETAIL:  Key (revision_type)=(General Revision) is not present in table "revision_type".`

Reply:
This happens because this test does a
```
truncate table cif.cif_user restart identity cascade;
```
which deletes all records in the database where the `created_by` or `updated_by` is set - including our `cif.revision_type` records. This won't be happening on our deployed instances since a superuser runs the script, but this is not the case when we run tests locally.

Instead, we should leave the `cif_user` table alone since it already contains basic users. If you need to control exactly what is in there you can update its values like this 
```
insert into cif.cif_user(id, uuid, etc...)
  overriding system value
  values (1, '11111111-1111-1111-1111-111111111111', etc...);
```

Reply:
We would want to refactor this, otherwise we don't test that our records in `revision_type` are inserted properly, and rely on extra code instead. this is also the case for the `cif.change_status` insert in this test